sopel/module.py

```diff
@@ -8,4 +8,38 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from sopel.plugin import *  # noqa
+# Import everything from sopel.plugin at the time of replacement.
+# Everything new from this point on must *not* leak here.
+# Therefore, don't add anything to this import list. Ever.
+from sopel.plugin import (  # noqa
```

Comment:
This is the source of LGTM's unused-import complaints, but it _shouldn't_ show up once the PR is merged.

I'm not a huge fan of LGTM's behavior around alert suppression in PRs (which is that it ignores suppression comments and shows all alerts Just In Case, making it impossible to verify that suppression comments _work_ as intended), but the documentation says `# noqa` works like `# lgtm`, so… I'll fix it if it somehow doesn't work after all following merge.